DC-SLE-RT-hardwaretesting

```diff
@@ -4,7 +4,7 @@
 ##
 ## Basics
 MAIN="MAIN.SLERT.xml"
-ROOTID=art-slert-hwtest
+ROOTID=article-slert-hardwaretest
```

Comment:
For the other products we also removed the product abbreviations from the file names and root IDs  because Stefan argued that those show up in the first part of the doc URL anyway.

Reply:
Thanks for the hint. I've made it consistent and used `article-hardware-testing`, also for the DC file (testing instead of test).